app.json

```diff
@@ -14,18 +15,13 @@
     }
   },
   "addons": [
-    "heroku-postgresql",
-    "papertrail"
-  ],
-  "buildpacks": [
-    {
-      "url": "https://github.com/heroku/heroku-buildpack-python"
-    },
     {
-      "url": "https://github.com/heroku/heroku-buildpack-apt"
+      "plan": "heroku-postgresql",
+      "option": {
+        "version": "11"
```

Comment:
I would omit the version here, so we use the latest supported by Heroku by default.

Reply:
Will handle this in https://github.com/niteoweb/pyramid-realworld-example-app/issues/86

Reply:
Will handle this in https://github.com/niteoweb/pyramid-realworld-example-app/issues/86

Reply:
Nope.. it's not postgres version related. Neither `psycopg2-binary`, `pyramid-deferred-sqla`
nor `zope.sqlalchemy`.

Reply:
Found ans in https://dba.stackexchange.com/questions/146135/psql-9-5-gen-random-uuid-not-working. Need to `CREATE EXTENSION pgcrypto;`